### DISCUSSION TITLE
Welcome to randomForestSRC Discussions!

### DISCUSSION THREAD

**Discussion by kogalur** [kogalur/randomForestSRC#108] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jianxinwang** [comment]
I'm a bioinformatician working in a cancer research institute. I really like this wonderful tool and want to learn more about it.

**Comment by jianxinwang** [comment]
I'm new to this tool. I have run through my data (cancer survival data). I'm trying to use gene expression data to predict survival time/probability. My input data contains survival in months and status (1, 0). After training a model (with this command obj.src <- rfsrc(Surv(Months.PFS, Statue) ~ ., dtaForRF, ntree = 1000, importance = TRUE, mtry = 420, node.size = 5, na.action = "na.impute"))I plotted a tree (plot(get.tree(obj.src, 3)) like below. I have 50 observations and 2500 features (genes). I'm wondering what the values in the leaf node means. Please help. Thanks! ![image](https://user-images.githubusercontent.com/11527387/145436477-caf3384b-7477-4868-a19e-5a311713c478.png)

**Comment by jianxinwang** [comment]
May be they are the risk scores? If so, I wonder how they are calculated.

**Comment by ishwaran** [comment]
The `get.tree` help file explains clearly what is being plotted in the terminal nodes.  

"Terminal nodes are highlighted by color and display the sample size and predicted value.   The predicted value displayed is as follows:"

 4. For survival, the choices are:

            • Mortality (‘mort’).

            • Relative frequency of mortality (‘rel.freq’).

            • Predicted survival (‘surv’), where the predicted survival
              is for the time point specified using ‘time’ (the default
              is the median follow up time).


Mortality is also defined in the help file.  However, you might want to look at the RSF vignette which could prove useful for other information.  https://luminwin.github.io/randomForestSRC/articles/survival.html

**Comment by rstrawderman** [comment]
Hi - I am wondering if it is possible / difficult to extend the Mahalonobis splitting function used for multivariate regression so that a user can input a fixed scale matrix instead of using the dynamically recomputed sample covariance? I think this would be a very useful extension of the current code.
